### ISSUE TITLE
"Delete this member" button is missing an icon

### ISSUE THREAD

**Issue by seldo** [npm/newww#1806]
"Actions" alt text is visible where there should be a button.

![screenshot 2016-01-27 11 26 20](https://cloud.githubusercontent.com/assets/185893/12625464/d22b88c6-c4e8-11e5-9dbf-aa9c07016375.png)

**Comment by jefflembeck** [comment]
This is due to the CSP blocking the JS that inserts the appropriate svg.

Let's go ahead and get around this by just accessibly placing an icon there instead of using the SVG route.

**Comment by stubbornella** [comment]
I opened two stories to our backlog for this work. One to add the icon and center it, and another to not offer the option to delete super admins.
